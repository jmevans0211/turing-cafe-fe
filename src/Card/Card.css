.card {
  background-color: white;
  border: 1px teal solid;
  color: teal;
  min-width: 200px;
}