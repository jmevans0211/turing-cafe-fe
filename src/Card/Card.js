import React from 'react';
import './Card.css'

const Card = props => {
  return(
    <div className="card"> 
      <h4>{props.name}</h4>
      <p>{props.date}</p>
      <p>{props.time}</p>
      <p>Number of Guests: {props.number}</p>
    </div>
  )

}

export default Card;