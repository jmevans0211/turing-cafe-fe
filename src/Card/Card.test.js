import React from 'react';
import { shallow } from 'enzyme'
import Card from './Card';

describe('Card', () => {
  let wrapper
  // const mockFn = jest.fn();

  beforeEach(() => {
    wrapper = shallow(<Card
      id = "1"
      name = "Jeannie"
      date = "6/30"
      time = "7:30"
      number = "3"
    />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

 })