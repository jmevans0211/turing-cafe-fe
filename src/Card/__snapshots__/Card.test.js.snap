// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<div
  className="card"
>
  <h4>
    Jeannie
  </h4>
  <p>
    6/30
  </p>
  <p>
    7:30
  </p>
  <p>
    Number of Guests: 
    3
  </p>
</div>
`;
